### PR TITLE
Added D3nii

### PR DESCRIPTION
**Pull request by D3nii**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/D3nii/D3nii.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Danyal Jamil
+institution: FAST NUCES Lahore
+profile_pic: Menoice.jpeg
```
Comment (Mohammed-Shoaib) [inline _data/D3nii/D3nii.md:4]: Incorrect file name, please do one of the following:
1. Change `Menoice.jpeg` to `D3nii.jpg ` in the markdown file.
2. Rename `D3nii.jpg ` to `Menoice.jpeg`.
I'd suggest you make the change in the markdown file!

**Comment by Abhishekds94** [comment]
> Hey @D3nii
> 
> image

Hey mate,
You are supposed to check all the files properly and only then need to approve. Please don't confuse others and be the reason for someone's loss due to your mistake.

**Comment by Mohammed-Shoaib** [comment]
@Abhishekds94 I agree, sometimes you can make a mistake but over here I had clearly suggested for changes. It seems like he's commenting when it's not required, look at [this](https://github.com/education/graduation/pull/4214#issuecomment-635797609) comment for example.

File: _data/D3nii/D3nii.md (hunk 1 above)
Comment (D3nii) [reply _data/D3nii/D3nii.md]: I changed it.

File: _data/D3nii/D3nii.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Danyal Jamil
+institution: FAST NUCES Lahore
+profile_pic: D3nii.jpeg
```
Comment (Mohammed-Shoaib) [inline _data/D3nii/D3nii.md:4]: Incorrect file name, please do one of the following:
1. Change `D3nii.jpeg` to `D3nii.jpg` in the markdown file.
2. Rename `D3nii.jpg` to `D3nii.jpeg`.
I'd suggest you make the change in the markdown file!

Comment (D3nii) [reply _data/D3nii/D3nii.md]: Thanks alot for the guidance. I did it!

**Comment by D3nii** [comment]
> Hey @D3nii,
> 
> Thanks for fixing the image extension! Your pull request looks good 👌
> 
> I confirm the following is correct:
> 
> 1. Checklist is complete. ✔️
> 2. Pull request only modified submission files (2 files) 📂
> 3. Profile picture is squared and has at least 544 pixels 🔳
> 
> I wish you all the best in your future endeavors! 🦄
> 
> Best,
> Mohammed Shoaib

Okay, so, what now, just wait?

**Comment by Mohammed-Shoaib** [comment]
Hey @D3nii,

Your pull request is perfect! :confetti_ball: I approved the changes. :+1:

However, I cannot merge it as I do not have write access. You'll have to wait for a reviewer with _write_ access to merge your pull request.

Till then sit back, relax, and enjoy :relaxed: 

Best,
Mohammed Shoaib

**Comment by Abhishekds94** [comment]
> @Abhishekds94 I agree, sometimes you can make a mistake but over here I had clearly suggested for changes. It seems like he's commenting when it's not required, look at [this](https://github.com/education/graduation/pull/4214#issuecomment-635797609) comment for example. 😕

I know right?
I do agree that we all have learned (and are still learning) many things over the past few days but as per my observation, none of us did this kind of review.

**Pull request merged**